### DISCUSSION TITLE
cannot run chrome os in 11th CPU (lakefield  i5-L16G7)

### DISCUSSION THREAD

**Discussion by muns5965** [sebanc/brunch#1050] [Q&A]
I use Galaxybook S (intel i5-L16G7)   <= 11th lakefield 
https://ark.intel.com/content/www/us/en/ark/products/202777/intel-core-i5-l16g7-processor-4m-cache-up-to-3-0ghz.html

I use "volteer" recovery image
and 
pressed "e" in grub menu, and
changed  "/kernel" to "/kernel-5.10" 
and pressed "F10" or "ctrl + x"  

but only brunch install ing  and reboot and brunch install . repeat..
 chrome os cannot boot  

Help me.

**Comment by kerta1n** [comment]
Hello, please consider that 5.10 is an in-development kernel and isn't fully finished yet. You will want to stick with the 5.4 kernel as that is the most recent but stable kernel released. At this point, you might just want to re-start your installation as it isn't worth trying to edit the existing GRUB config file.

**Comment by BrunoGM** [comment]
Have you ever found a solution for that? I get black screen and reboot when launching brunch with the L16G7
